hanadb_exporter/main.py

```diff
@@ -79,16 +83,16 @@ def handle_exception(*exc_info): # pragma: no cover
     sys.excepthook = handle_exception
 
 
-def find_metrics_file():
+def lookup_etc_folder(default_paths):
```

Comment:
Maybe better `config_files_path` rather than `default_paths`. It it's a variable it doesn't have much sense to use the `default` name there